src/components/formly.common.component.ts

```diff
@@ -36,19 +37,20 @@ export class FormlyCommon {
     this.ps.Stream.emit(this.form);
   }
 
-  isHidden() {
-    return this.hide;
+  @Input()
+  public get hide() {
+    return this._hide;
   }
-  setHidden(cond: boolean) {
-    this.hide = cond;
+  public set hide(value: boolean) {
+    this._hide = value;
 
-    this.elem.nativeElement.style.display = cond ? "none" : "";
```

Comment:
Please review if this is best way to use renderer so that it can run on server side.